Ginger/Ginger/Variables/VariableEditPage.xaml.cs

```diff
@@ -336,7 +336,8 @@ private void resetBtn_Click(object sender, RoutedEventArgs e)
 
         private void AutoValueBtn_Click(object sender, RoutedEventArgs e)
         {
-            mVariable.GenerateAutoValue();
+            string errorMsg = string.Empty;
+            mVariable.GenerateAutoValue(ref errorMsg);
```

Comment:
here we not supposedd to show message to user in case error exist in the errorMsg?